test/get_authors_test.rb

```diff
@@ -53,6 +48,17 @@ def test_markdown_center
     assert_match %Q|{:center: style="text-align: center"}\n#{text}\n{:center}|, Markdown.center(text)
   end
 
+  def test_git_log
+    method_output = Git.log File.join(@subdir, 'README.md')
+    assert_equal "Your Name;you@example.com", "#{method_output}"
+  end
+
+  def test_latest_author_email_is_used
+    output = "Your Name;you@example.com\nYour Name;me@example.com"
+    git_log = Git::Log.new output
+    assert_equal({"Your Name"=>"you@example.com"}, git_log.author_email)
```

Comment:
Great!
Exactly what I wanted. Could you also add tests for multiple different authors?

Reply:
Hmm, what should be tested there? If it can handle multiple authors with different emails as well?

Reply:
Yes, exactly. Just the standard usage. Additionally, for example:

  - three of them, with the one in the middle  - different from the other one.
  - six different people, while the six of them are different

The question is: what should happen, when you have e.g. 10 commits and last 5 are from a single person.

Reply:
I think it would just write the person, I'll deeper look into that and the tests as well but it'll take me some time.

Reply:
Sure, that's just an inspiration for another PR.